### PR TITLE
docs: add initial contributors guidelines

### PR DESCRIPTION
**Pull request by mainawycliffe**
Adding initial contributors guidelines.

### CODE REVIEW

**Comment by mainawycliffe** [comment]
https://github.com/zino-app/graphql-flutter/issues/262

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+- [Question or Problem?](#question)
+- [Issues and Bugs](#issue)
+- [Feature Requests](#feature)
+- [Submission Guidelines](#submit)
+- [Coding Rules](#rules)
+- [Commit Message Guidelines](#commit)
+
+## <a name="question"></a> Got a Question or Problem?
+
+Do not open issues for general support questions as we want to keep GitHub issues for bug reports and feature requests. You've got much better chances of getting your question answered on [Stack Overflow][stackoverflow] where the questions should be tagged with tag `flutter`.
```
Comment (micimize) [inline CONTRIBUTING.md]: tagged with [`graphql` and `flutter`](https://stackoverflow.com/questions/tagged/graphql+flutter)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+- **perf**: A code change that improves performance
+- **refactor**: A code change that neither fixes a bug nor adds a feature
+- **style**: Changes that do not affect the meaning of the code (white-space, formatting, linting etc)
+- **test**: Adding missing tests or correcting existing tests
+
+### Scope
+
+The scope should be the name of the npm package affected (as perceived by the person reading the changelog generated from commit messages.
+
+The following is the list of supported scopes:
```
Comment (micimize) [inline CONTRIBUTING.md]: I didn't realize this 😅

**Pull request merged**